tools/modules/system/manage_dtoverlays.sh

```
@@ -74,6 +77,10 @@ function manage_dtoverlays () {
 			0)
 				changes="true"
 				newoverlays=$(echo $selection | sed 's/"//g')
+				# handle case where overlay_prefix is part of overlay name
+				newoverlays=$(echo $newoverlays | sed "s/^$overlay_prefix"-"//" \
```

Shellcheck (0.10.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)